litellm/utils.py

```diff
@@ -14,6 +14,7 @@
 import litellm, openai
 import itertools
 import random, uuid, requests
+from functools import wraps
```

Comment:
do we need to add `functools` as a depedancy?

Reply:
as far as I know, It is a standard library which doesn't need to be included as a external dependancy